composer.json

```diff
@@ -13,8 +13,9 @@
     ],
     "require": {
         "php": ">=8.1",
-        "symfony/framework-bundle": "^6.4|^7.0",
+        "symfony/framework-bundle": "^6.4.1|^7.0",
```

Comment:
To include the following fix [https://github.com/symfony/symfony/pull/52852](https://github.com/symfony/symfony/pull/52852) for synfony serializer